src/main/java/cloud/fogbow/ras/core/cloudconnector/LocalCloudConnector.java

```diff
@@ -47,18 +47,20 @@
     private SecurityRulePlugin securityRulePlugin;
     private GenericRequestPlugin genericRequestPlugin;
 
+    private boolean shouldAuditRequests;
+
     public LocalCloudConnector(String cloudName) {
         InteroperabilityPluginInstantiator instantiator = new InteroperabilityPluginInstantiator(cloudName);
```

Comment:
As per previous comment, there is no need to send cloudName in the constructor.

Reply:
We do need the cloud name in this case because an instance of this class points to a specific cloud configured by the user.

Reply:
Right